README.md

```diff
@@ -20,30 +20,55 @@ dotnet tool install -g gitsmimesign
 
 The following is how to install with GIT versions 2.19 or newer.
 
+#### Configure globally
+
+```Batchfile
+git config --global gpg.X.509.program gitsmimesign
+git config --global gpg.format X.509
+```
+
+If you want to always use sign commits by default set:
+
+```Batchfile
+git config --global commit.gpgsign true
+```
+
 #### Configure for local repository only
 
 To configure only a local repository to use the `gitsmimesign`.
 
 ```Batchfile
 cd \to\path\of\repository
-git config --local gpg.x509.program gitsmimesign
-git config --local gpg.format x509
+git config --local gpg.X.509.program gitsmimesign
```

Comment:
These need to be x509 due to the it being a git command.

Reply:
Noted. Fixed.